protocols/6d3eda/6d3eda.ot2.apiv2.py

```diff
@@ -0,0 +1,22 @@
+metadata = {"apiLevel": "2.5"}
+
+
+def run(ctx):
+    # Add tip tracking to this protocol
+    count, volume, input_labware, output_labware = get_values(  # noqa: F821
+            'sample_count', 'volume', 'input_labware', 'output_labware')
+    sample_plates = [ctx.load_labware(input_labware, '2')]
+    target_plates = [ctx.load_labware(output_labware, '5')]
+    tip_racks = [ctx.load_labware('opentrons_96_filtertiprack_200ul', '1')]
+    if count > 24:
+        sample_plates.append(ctx.load_labware(input_labware, '3'))
+        target_plates.append(ctx.load_labware(output_labware, '6'))
+
+    # Variable pipettes?
+    p300s = ctx.load_instrument(
+            'p300_single_gen2', "right", tip_racks=tip_racks)
+
+    for i, sample_plate in enumerate(sample_plates):
+        for well_num, well in enumerate(sample_plate.wells()[:count-(i*24)]):
+            p300s.transfer(volume, well, target_plates[i].wells()[well_num],
```

Comment:
@Koeng101 if you'd like to add tip tracking, a simple version might look like this:

> try:
>      p300s.pick_up_tip()
> except protocol_api.labware.OutOfTipsError:
>     protocol.pause("Replace the tips")
>     p300s.reset_tipracks()
>     p300s.pick_up_tip()

If you put this in the for loop before `p300s.transfer(...` and remove the `new_tip='always')`, you should be good

Reply:
Added with commit f792070

Reply:
One thing to note, about this is that the proocol_api in the `protocol_api.labware.OutOfTipsError`: should be imported at the top of the file

Reply:
For example, [see this](https://github.com/Opentrons/Protocols/blob/develop/protocols/21e4d8-pt2/21e4d8-pt2.ot2.apiv2.py)